client/src/components/CreatePetModal.jsx

```diff
@@ -170,11 +195,25 @@ export function CreatePetModal() {
                   disabled={!species}
                   required
                 >
-                  {breedList.map((breed) => (
-                    <Option key={breed.id} name={breed.name} value={breed.name}>
-                      {breed.name}
-                    </Option>
-                  ))}
+                  {species == "cat"
+                    ? catBreedList.map((breed) => (
```

Comment:
consider adding an id or aria-label to make it easier for screen readers to understand your code

Reply:
Will do! I am honestly having some trouble rendering this specific dropdown menu, as using map has caused some unexpected behavior when selecting options, so this will probably be completely refactored. I will include an id or aria-label in the refactor.